ng2-components/ng2-activiti-tasklist/src/components/activiti-start-task.component.ts

```diff
@@ -37,17 +37,21 @@ export class ActivitiStartTaskButton {
     onSuccess: EventEmitter<any> = new EventEmitter<any>();
 
     @Output()
-    error: EventEmitter<any> = new EventEmitter<any>();
+    cancel: EventEmitter<void> = new EventEmitter<void>();
 
-    @ViewChild('dialog')
-    dialog: any;
-
-    forms: Form [];
+    @Output()
+    error: EventEmitter<any> = new EventEmitter<any>();
 
-    formId: string = null;
+    @Input()
+    people: User [] = [];
 
     name: string;
+    forms: Form [];
```

Comment:
Create a single Model that contains all these properties. 
Something like 

export class StartTaskModel {
    name: string;
    description: string;
    forms: Form [];
…..